modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/management_bar/controls/filters/ActiveFiltersBar.js

```diff
@@ -24,17 +30,50 @@ function ActiveFiltersBar({disabled}) {
 				selectedData: undefined,
 			})),
 		});
+
+		if (Liferay.FeatureFlags['LPD-52212']) {
+			onSearch({query: ''});
+		}
 	};
 
 	const activeFilters = filters.filter((filter) => filter.active);
 
-	return activeFilters.length ? (
-		<div className="management-bar management-bar-light navbar navbar-expand-md">
+	return activeFilters.length ||
+		(Liferay.FeatureFlags['LPD-52212'] && isSearchActive()) ? (
+		<div
+			className="management-bar management-bar-light navbar navbar-expand-md"
+			data-qa-id="activeFiltersToolbar"
+		>
 			<div className="container-fluid">
-				<nav className="mb-0 py-3 subnav-tbar subnav-tbar-light subnav-tbar-primary w-100">
+				<nav className="subnav-tbar subnav-tbar-light w-100">
 					<ul className="tbar-nav">
 						<li className="p-0 tbar-item tbar-item-expand">
 							<div className="tbar-section">
+								{Liferay.FeatureFlags['LPD-52212'] &&
+									(dataLoading ? (
+										<span>
+											{Liferay.Language.get(
+												'requesting-results-for-colon'
+											)}
+										</span>
+									) : (
```

Comment:
`dataLoading` is tied to every type of loading in FDS. So, when user changes pagination page, sorts, deletes an item etc, user will see this loading effect in filters bar. That may be unexpected noise in UI. See movie below. Are we ok with this from design / product perspective? If not, it would be possible to implement separate loading state, just for search and filters.

Also, in general, loading behavior indicated with a label is quite unusual for DXP, we consistently have spinner icon as the loading indicator. Is this ok from design / product perspective? 

@ugeortiz @dsanz @herrardo  

https://github.com/user-attachments/assets/e3ae8005-991f-4345-8f28-f26f91d2d071

Reply:
That's a good point I didn't consider. That does seem strange seeing that messaging when the total results have already been found. In the meantime, I'll think through a way of preventing this, or like you suggested with a separate loading state.